eli5/formatters/utils.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+
+def replace_starting_trailing_spaces(s, replacer):
```

Comment:
Yeah, I see, that could help indeed. I think having spaces both inside and outside should be rather rare, but the user can notice how space is marked when working on bigram features, I'll check how it looks like.
And also let's add a title to that span with some message that tells that this is a space here, like "A space symbol"